easybuild/easyconfigs/b/BirdNET/BirdNET-20201214-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,105 @@
+easyblock = 'Tarball'
+
+name = 'BirdNET'
+version = '20201214'
+versionsuffix = '-Python-%(pyver)s'
+local_commit = '022a6d5'
+
+homepage = 'https://birdnet.cornell.edu/'
+description = """BirdNET is a research platform that aims at recognizing birds by sound at scale.
+We support various hardware and operating systems such as Arduino microcontrollers, the Raspberry Pi, 
+smartphones, web browsers, workstation PCs, and even cloud services. BirdNET is a citizen science 
+platform as well as an analysis software for extremely large collections of audio.
+BirdNET aims to provide innovative tools for conservationists, biologists, and birders alike."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+github_account = 'kahst'
+sources = [
+    {
+        'source_urls': [GITHUB_SOURCE],
+        'download_filename': '%s.tar.gz' % local_commit,
+        'filename': SOURCE_TAR_GZ,
+    },
+    {
+        'source_urls': ['https://tuc.cloud/index.php/s/m9smX4FkqmJaxLW/'],
+        'download_filename': 'download',
+        'filename': 'BirdNET_Soundscape_Model.pkl',
+        'extract_cmd': 'cp %s %(builddir)s',
+    },
+]
+patches = ['BirdNET-20201214_add-model-path-envvar.patch']
+
+checksums = [
+    '3a7a5f562bfdd55138603847d206125328f02a25df5e86278d2fb35242bb1c1d',  # BirdNET-20201214.tar.gz
+    '7db0b2fe57da002aff62fe1aecf56b16065c2b699dcbee1d8d7e07a42719b3fe',  # BirdNET_Soundscape_Model.pkl
+    '33cbddca35223eb078ed62e7eb1748930225dfe43a40b9d093ae0d6d18a1344b',  # BirdNET-20201214_add-model-path-envvar.patch
+]
+
+dependencies = [
+    ('FFmpeg', '4.2.1'),
+    ('libsndfile', '1.0.28'),
+    ('Python', '3.7.4'),
+    ('libgpuarray', '0.7.6', versionsuffix),
+    ('numba', '0.47.0', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Theano', '1.0.4', versionsuffix),
+]
+
+postinstallcmds = [
+    'chmod +x %(installdir)s/analyze.py',
+    'cp %(builddir)s/BirdNET_Soundscape_Model.pkl %(installdir)s/model/',
+]
+
+fix_python_shebang_for = ['analyze.py']
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
```

Comment:
You should be able to drop this line, since `PythonPackage` knows how to download from PyPI...